debian/install

```diff
@@ -1,2 +1,4 @@
 debian/homeserver.yaml etc/matrix-synapse
 debian/log.yaml etc/matrix-synapse
+debian/matrix-synapse.list etc/apt/sources.list.d
+debian/matrix-synapse.gpg etc/apt/trusted.gpg.d
```

Comment:
Ugh.  I hate it when packages stuff things in /etc/apt.  Please, no.

Reply:
It's also done by *debian-ports-archive-keyring* and *debian-archive-keyring*. What's wrong with using these directories?

```
% dpkg -L debian-ports-archive-keyring debian-archive-keyring G etc/apt/
/etc/apt/trusted.gpg.d
/etc/apt/trusted.gpg.d/debian-ports-archive-2017.gpg
/etc/apt/trusted.gpg.d/debian-ports-archive-2018.gpg
/etc/apt/trusted.gpg.d/debian-ports-archive-2019.gpg
/etc/apt/trusted.gpg.d
/etc/apt/trusted.gpg.d/debian-archive-jessie-automatic.gpg
/etc/apt/trusted.gpg.d/debian-archive-jessie-security-automatic.gpg
/etc/apt/trusted.gpg.d/debian-archive-jessie-stable.gpg
/etc/apt/trusted.gpg.d/debian-archive-stretch-automatic.gpg
/etc/apt/trusted.gpg.d/debian-archive-stretch-security-automatic.gpg
/etc/apt/trusted.gpg.d/debian-archive-stretch-stable.gpg
/etc/apt/trusted.gpg.d/debian-archive-wheezy-automatic.gpg
/etc/apt/trusted.gpg.d/debian-archive-wheezy-stable.gpg
```

Reply:
For reference: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=66325